### DISCUSSION TITLE
Drag files into application pins to launch an app and make it open a specific file

### DISCUSSION THREAD

**Discussion by sevban** [lawand-dot-io/taskbar#80] [Feature requests]
We can do this with Dock. For example you can drag an image on Firefox and Drop it into the Safari on the Dock and Safari will display that image.

**Comment by lawand-dot-io** [comment]
Thanks for the request. This should in theory already work with Taskbar, but maybe I should tweak it. Can you maybe do a quick screen recording showing an example of what you mean? thanks
